openssl/pbpal_openssl.c

```diff
@@ -111,6 +115,14 @@ static int pal_init(void)
         if (locks_setup()) {
             return -1;
         }
+        #else // OPENSSL_VERSION_NUMBER >= 0x10100000L
+        OPENSSL_init_ssl(OPENSSL_INIT_LOAD_SSL_STRINGS, NULL);
+        OPENSSL_init_crypto(
+                OPENSSL_INIT_ADD_ALL_CIPHERS | OPENSSL_INIT_ADD_ALL_DIGESTS,
+                NULL
+        );
```

Comment:
If we want to set up this manually, are there any reasons why other default options wasn't included?
`OPENSSL_INIT_LOAD_CRYPTO_STRINGS ` – this one will include error strings, for example.

Docs has more flags marked as default.